### ISSUE TITLE
SOGo as Roundcube+ownCloud Replacement?

### ISSUE THREAD

**Issue by aspdye** [mail-in-a-box/mailinabox#644]
Groupware with native ActiveSync support, like Mailcow (mailcow.email) is using it?
https://github.com/andryyy/mailcow/tree/sogo-master
http://sogo.nu/

**Comment by nstanke** [comment]
This would be really cool, especially MAPI support. Unfortunately replacing  Owncloud, Roundcube & Z-Push with SOGo & OpenChange would require two "major" changes. 

* Replacing our SQlite storage with a database server (PostgreSQL/MySQL)
* Using a LDAP server (#119)

Further this setup would require a minimum of 2048MB RAM[1]. Please also see #514 for general discussion on a CalDav/CardDav replacement.

[1] http://www.sogo.nu/files/docs/SOGo%20Installation%20Guide.pdf

**Comment by aspdye** [comment]
I think the Database Server shouldnt be a Problem. Same with the LDAP Stuff. This could make MiaB even better. Problems could be the Update (Migration from SQLite to MySQL and LDAP). 
The 2 GB of needed RAM are in my opionion the reason why @joshdata will not do this. A machine with 2 GB RAM costs 100% more and the users have to change their Droplet or VM. 
Maybe @joshdata could make a Classic and Groupware Branch. Classic could only contain a Mailserver Without ownCloud and Zpush stuff (only Roundcube), and Groupware could contain SOGo with Active Sync, Calendar and Contacts ;)

**Comment by JoshData** [comment]
I'm definitely not creating two branches! I barely have enough time to worry about the first branch.

Installing MySQL/Postgres and a LDAP server are both no-goes. Remember, we're trying to make maintenance of this project _easier_ by moving away from ownCloud, not harder!

**Comment by arodier** [comment]
Hello Josh et al,

If this can help, I have started to publish and improved the scripts I used to deploy my personal mail server at home. I always used LDAP for SSO.

The whole installation is automated through Ansible, but the LDAP part is easy:
https://github.com/progmaticltd/homebox/blob/dev-arodier/install/playbooks/roles/ldap/tasks/main.yml

Perhaps I will add a reference to your project in my readme file, this is a fantastic work.

Any question welcome.
